src/Language/PureScript/Errors.hs

```diff
@@ -535,25 +539,25 @@ prettyPrintSingleError full level e = prettyPrintErrorMessage . positionHintsFir
                                           , indent $ prettyPrintValue expr
                                           ]) binding
     renderSimpleErrorMessage (TypesDoNotUnify t1 t2)
-      = paras [ line "Cannot unify type"
+      = paras [ line "The type"
               , indent $ typeAsBox t1
-              , line "with type"
+              , line "cannot be made equal to the type"
               , indent $ typeAsBox t2
               ]
     renderSimpleErrorMessage (KindsDoNotUnify k1 k2) =
-      paras [ line "Cannot unify kind"
+      paras [ line "The kind"
             , indent $ line $ prettyPrintKind k1
-            , line "with kind"
+            , line "cannot be made equal to the kind"
             , indent $ line $ prettyPrintKind k2
             ]
     renderSimpleErrorMessage (ConstrainedTypeUnified t1 t2) =
-      paras [ line "Cannot unify constrained type"
+      paras [ line "The constrained type"
             , indent $ typeAsBox t1
-            , line "with type"
+            , line "cannot be made equal to the kind"
             , indent $ typeAsBox t2
             ]
     renderSimpleErrorMessage (OverlappingInstances nm ts (d : ds)) =
-      paras [ line "Overlapping instances found for"
+      paras [ line "Overlapping type class instances found for"
```

Comment:
Do we need to add a one line explanation of why this is bad perhaps? Or is that one for the wiki.